apps/web/playwright/lib/testUtils.ts

```diff
@@ -92,7 +92,9 @@ export async function selectFirstAvailableTimeSlotNextMonth(page: Page) {
   await page.waitForTimeout(1000);
   // TODO: Find out why the first day is always booked on tests
   await page.locator('[data-testid="day"][data-disabled="false"]').nth(1).click();
-  await page.locator('[data-testid="time"][data-disabled="false"]').nth(0).click();
+  // Changed to dblclick because of some flakiness when click a button animated
+  // It is said playwright should wait for animations to end, but it doesn't seem to be the case on this one
+  await page.locator('[data-testid="time"]').nth(0).dblclick();
```

Comment:
This should be improved, but I was getting error while using new booker looks like playwright can get it's right X,Y position because it's animated and doing one single click didn't work.
@joeauyeung @zomars

Reply:
Can we artificially wait a few seconds or wait for an element to be rendered?